test/common/integration/client_integration_test.cc

```diff
@@ -211,54 +236,31 @@ TEST_P(ClientIntegrationTest, BasicNon2xx) {
           Http::TestResponseHeaderMapImpl({{":status", "503"}, {"content-length", "0"}})));
 
   envoy_stream_t stream = 1;
-  // Setup bridge_callbacks to handle the response.
-  envoy_http_callbacks bridge_callbacks;
-  ConditionalInitializer terminal_callback;
-  callbacks_called cc = {0, 0, 0, 0, 0, &terminal_callback};
-  bridge_callbacks.context = &cc;
-  bridge_callbacks.on_headers = [](envoy_headers c_headers, bool end_stream,
-                                   void* context) -> void* {
-    EXPECT_TRUE(end_stream);
-    Http::ResponseHeaderMapPtr response_headers = toResponseHeaders(c_headers);
-    EXPECT_EQ(response_headers->Status()->value().getStringView(), "503");
-    callbacks_called* cc = static_cast<callbacks_called*>(context);
-    cc->on_headers_calls++;
-    return nullptr;
-  };
-  bridge_callbacks.on_complete = [](void* context) -> void* {
-    callbacks_called* cc = static_cast<callbacks_called*>(context);
-    cc->on_complete_calls++;
-    cc->terminal_callback->setReady();
-    return nullptr;
-  };
-  bridge_callbacks.on_error = [](envoy_error error, void* context) -> void* {
-    error.message.release(error.message.context);
-    callbacks_called* cc = static_cast<callbacks_called*>(context);
-    cc->on_error_calls++;
-    return nullptr;
-  };
-
+  // Setup bridge_callbacks_ to handle the response.
```

Comment:
Is this comment still needed now that much of the setup logic is in Initialize()?